examples/counter/src/lib.rs

```diff
@@ -0,0 +1,38 @@
+use rock_n_rollup::{
+    core::{Application, Runtime},
+    plugins::{
+        database::{Database, Json},
+        logger::Logger,
+    },
+    services::external::{External, FromExternal},
+};
+
+pub struct Decrement {}
```

Comment:
Nice, it's the way to declare action from the user.

In the future (not in this MR), we should think a way to have something more user friendly, it would be awesome if the user does not have to specify the encoding